src/HassClient/Client/HassClient.cs

```diff
@@ -793,33 +787,19 @@ public async Task<bool> SubscribeToEvents(EventType eventType = EventType.All)
 
             if (eventType != EventType.All)
             {
-                switch (eventType)
+                command.EventType = eventType switch

```

Comment:
We could refactor this by adding an attribute `EventTypeAttribute` with a string, and place on the enum members. Then get the attribute and the underlying string.

Reply:
If so, do it in separate PR.